samcli/lib/init/__init__.py

```diff
@@ -104,3 +109,22 @@ def generate_project(
         raise InvalidLocationError(template=params["template"]) from e
     except CookiecutterException as e:
         raise GenerateProjectFailedError(project=name, provider_error=e) from e
+
+
+def _fix_gradlew_line_ending(path):
```

Comment:
Should we make this more generic so it can accept multiple paths? In case there are other Unix-only files that might be messed up by Windows.

Reply:
Updated.